### PR TITLE
feat: Introduce the new Price Filter component

### LINKED ISSUE #1294: [5.2] Filter price available inventory
## 🎉 Description

Enhance the current price filter for LANDs and add a map showing the density of NFTs by price.

* [ ] 🐛 This is a bug report.
* [X] 📈 This is a feature request.

### PR DESCRIPTION
**Pull request by juanmahidalgo**
Closes #1294

### CODE REVIEW

File: webapp/src/components/PriceChart/PriceChart.css [hunk 1]
```diff
@@ -0,0 +1,197 @@
+.price-filter.range-field {
```
Comment (juanmahidalgo) [inline webapp/src/components/PriceChart/PriceChart.css:1]: This whole module is moving out to the UI as well. PR: https://github.com/decentraland/ui/pull/283

File: webapp/src/components/AssetFilters/PriceFilter/PriceFilter.tsx [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+import { ethers } from 'ethers'
+import { Box, useTabletAndBelowMediaQuery } from 'decentraland-ui'
 import { Network } from '@dcl/schemas/dist/dapps/network'
 import { t } from 'decentraland-dapps/dist/modules/translation/utils'
 import { getPriceLabel } from '../../../utils/filters'
+import { nftAPI } from '../../../modules/vendor/decentraland'
+import { Section } from '../../../modules/vendor/routing/types'
+import { getChartUpperBound, sectionToPriceFilterOptions } from './utils'
+import { Props } from './PriceFilter.types'
+import { PriceChart } from '../../PriceChart/PriceChart'
```
Comment (kevinszuchet) [inline webapp/src/components/AssetFilters/PriceFilter/PriceFilter.tsx:11]: Would you move this import two rows above? 🙏

Comment (juanmahidalgo) [reply webapp/src/components/AssetFilters/PriceFilter/PriceFilter.tsx:9]: done! Also, some changes introduced in the PriceFilter component so it sends the NFTFilters (rarity, network, category, etc)

File: webapp/src/components/AssetFilters/PriceFilter/PriceFilter.container.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  getWearableGenders
+} from '../../../modules/routing/selectors'
+import { getCategoryFromSection } from '../../../modules/routing/search'
+import { MapStateProps } from './PriceFilter.types'
+import { PriceFilter } from './PriceFilter'
+
+const mapState = (state: RootState): MapStateProps => {
+  const section = getSection(state)
+  return {
+    section: getSection(state),
```
Comment (kevinszuchet) [inline webapp/src/components/AssetFilters/PriceFilter/PriceFilter.container.tsx:20]:
```suggestion
    section,
```

File: webapp/src/config/env/dev.json [hunk 4]
```diff
@@ -1,11 +1,11 @@
 {
   "NETWORK": "goerli",
   "CHAIN_ID": "5",
-  "NFT_SERVER_URL": "https://nft-api.decentraland.zone/v1",
+  "NFT_SERVER_URL": "https://nft-api.decentraland.today/v1",
```
Comment (kevinszuchet) [inline webapp/src/config/env/dev.json:4]: Should we use .zone instead? (*)

File: webapp/src/config/env/dev.json [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
   "BUILDER_SERVER_URL": "https://builder-api.decentraland.zone/v1",
   "ATLAS_SERVER_URL": "https://api.decentraland.zone",
   "PEER_URL": "https://peer-ap1.decentraland.zone",
-  "SIGNATURES_SERVER_URL": "https://signatures-api.decentraland.zone/v1",
+  "SIGNATURES_SERVER_URL": "https://signatures-api.decentraland.today/v1",
```
Comment (kevinszuchet) [inline webapp/src/config/env/dev.json:8]: (*) Same here

File: webapp/src/config/env/dev.json (hunk 4 above)
Comment (juanmahidalgo) [reply webapp/src/config/env/dev.json:4]: sorry, I forgot to add a comment in the PR for this change, this is temporary, just to show the prod data in the Vercel preview. I'll revert this change before merging

File: webapp/src/config/env/dev.json (hunk 5 above)
Comment (juanmahidalgo) [reply webapp/src/config/env/dev.json:8]: Same answer as above, good catch tho!

File: webapp/src/components/AssetFilters/PriceFilter/PriceFilter.container.tsx (hunk 3 above)
Comment (juanmahidalgo) [reply webapp/src/components/AssetFilters/PriceFilter/PriceFilter.container.tsx]: done! good call :)

File: webapp/src/components/AssetFilters/AssetFilters.tsx [hunk 6]
```diff
@@ -124,14 +125,14 @@ export const AssetFilters = ({
     [category, section]
   )
 
-  if (isInLandSection) {
+  if (isInLandSection && isPriceFilterEnabled) {
```
Comment (meelrossi) [inline webapp/src/components/AssetFilters/AssetFilters.tsx:128]: this will remove the land status filter too if price is not enabled

Comment (juanmahidalgo) [reply webapp/src/components/AssetFilters/AssetFilters.tsx]: ohhhh nice catch, updated!

**Review by meelrossi — APPROVED** [review]

**Pull request merged**